drivers/windows/pcifilter/flt_dbg.h

```diff
@@ -0,0 +1,58 @@
+/***
+CHIPSEC: Platform Security Assessment Framework
+Copyright (c) 2010-2023, Intel Corporation

```

Comment:
New files should just have 2023 copyright dates.

Reply:
got